config/routes.rb

```diff
@@ -1,21 +1,58 @@
 Rails.application.routes.draw do
+  root "commodities#index"
+
   get '/home', to: "home#index"
+
   get '/search', to: "cars#search"
   get '/download_pdf', to: "cars#download_pdf"
+
   get 'commodities/home', to: "commodities#home", as: "commodity_home"
-  get 'customers/top_customers/:task', to: "customers#top_customers", as: "top_customers"
+
+  
   get 'orders/filtered_orders', to: "orders#filtered_orders", as: "filtered_orders"
   get 'order/search', to: "orders#search"
+  
+  get 'employee/email/search', to: "employees#search"
+  get 'employee/increment/:id', to: "employees#increment", as: "order_increment"
+  get 'employee/decrement/:id', to: "employees#decrement", as: "order_decrement"
+  get 'employee/all_employees', to: "employees#all_employees", as: "all_employees"
+  get 'employee/filtered_employees', to: "employees#filtered_employees", as: "filtered_employees"
+
```

Comment:
We can optimize this by using the `collection` and `member` - https://guides.rubyonrails.org/routing.html#adding-more-restful-actions